### DISCUSSION TITLE
why is loss reduced when distributed training

### DISCUSSION THREAD

**Discussion by LukeCho-8810** [open-mmlab/mmdetection#9167] [Q&A] [answered]
Hi, thanks for your great work.

I dont know why loss is reduced explicitly when distributed training. Is this not be done in pytorch ddp

https://github.com/open-mmlab/mmdetection/blob/ca11860f4f3c3ca2ce8340e2686eeaec05b29111/mmdet/models/detectors/base.py#L214

**Comment by jbwang1997** [comment, marked as answer]
Sorry for not getting your question. mmdetection is built on pytorch. We directly use pytorch DDP in distributed training.
